CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Version 0.4.1-dev (changes since 0.4.1 go here)
 
 ### Features
+* Added `skbio.tree.TreeNode.bifurcate` for converting multifurcating trees into bifurcating trees. [#896](https://github.com/biocore/scikit-bio/issues/896))
```

Comment:
The issue number doesn't render correctly, can you add an opening paren?

![screen shot 2016-02-17 at 11 21 32 am](https://cloud.githubusercontent.com/assets/1847232/13119904/a31325e0-d568-11e5-8f6e-b1822727b921.png)